docs: add instructions for first boot tui (#2141)
spectrocloud/librarium: docs/docs-content/clusters/edge/site-deployment/site-installation/edge-host-registration.md

---

To use an Edge host with a host cluster, you must first register it with Palette. A registration token in the user data

> is required to complete the registration process. You have two options to register the Edge host with Palette.

Reviewer: I would advise against counting the number of options here, since this introduces a variable that could be difficult to update in the future. Instead, you could go with something like:
> You have the following options to register the Edge host with Palette.

> is required to complete the registration process. You have two options to register the Edge host with Palette.

Author: ```suggestion
is required to complete the registration process. You have the following options to register the Edge host with Palette.
```

| **Method**          | **Description**                                                                                                                                                  | **Set up Effort** |
| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |

> | Auto Registration   | Edge hosts can automatically register with Palette by using a _Registration Token_. This method requires you to specify the registration token in the user data. | Low               |

Reviewer: ```suggestion
| Auto Registration   | Edge hosts can automatically register with Palette through a _Registration Token_. This method requires you to specify the registration token in the user data. | Low               |
```

Reviewer: I would change the wording here because, technically, this is not an active action, as Edge hosts passively use the token value the user specifies.

> | Manual Registration | You can manually enter a unique Edge host ID into Palette.                                                                                                       | Low               |

Reviewer: ```suggestion
| Manual Registration | You can manually enter a unique Edge host ID in Palette.                                                                                                       | Low               |
```

Reviewer: I'm a bit confused with the into, since it feels somehow very physical, like putting something into a container, you know? Would 'in' do the job?
